journal/src/main/scala/com/evolutiongaming/kafka/journal/Events.scala

```diff
@@ -28,7 +31,9 @@ object Events {
     codecs.choice(version1, version0, default)
   }
 
-  implicit def eventsToBytes[F[_] : FromAttempt]: ToBytes[F, Events] = ToBytes.fromEncoder
+  implicit def eventsToBytes[F[_] : FromAttempt](implicit jsonCodec: JsonCodec[Try]): ToBytes[F, Events] =
```

Comment:
Why whole `JsonCodec[Try]` is required here, rather than `JsonCodec.Encode[Try]`?

Reply:
Same: implemented in terms of `scodec.Codec`